views/components/BoatCompass/BoatCompass.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Image from "next/image";
+import { Grid, Paper } from "@mui/material";
+
+interface BoatCompassProps {
+    angle: number;
+}
+
+class BoatCompass extends React.Component<BoatCompassProps> {
+    render() {
+        const { angle } = this.props;
+
+        return (
+            <Paper elevation={0}>
+                <Grid
+                    container
+                    direction="row"
+                    justifyContent="center"
+                    alignItems="center"
+                    width={70}
+                    height={70}
+                >
+                    <Image
+                        src="/boat.png"
+                        width={50}
+                        height={50}
+                        alt="Boat Icon"
+                        style={{ transform: this._rotateBoatString(angle) }}
+                    />
```

Comment:
Do you think it would be useful to also include a outlined circle around the boat indicating the degrees?

Something like this:

![image](https://github.com/UBCSailbot/website/assets/59242538/0d0ad374-399a-49c0-b9ce-4385022dc269)
 

Ref: https://www.google.com/search?q=degree+circle+compass&tbm=isch&ved=2ahUKEwiF-YSMkbyEAxWiHzQIHXV3CLAQ2-cCegQIABAA&oq=degree+circle+compass&gs_lp=EgNpbWciFWRlZ3JlZSBjaXJjbGUgY29tcGFzczIFEAAYgAQyBhAAGAgYHkjFDVCbB1iCDXAAeACQAQCYAesBoAH2BaoBBTYuMS4xuAEDyAEA-AEBigILZ3dzLXdpei1pbWfCAgQQABgewgIGEAAYBRgewgIHEAAYgAQYGIgGAQ&sclient=img&ei=O8TVZcXvD6K_0PEP9e6hgAs&bih=782&biw=1554&hl=en#imgrc=gC-s3YZYhW8uxM

Reply:
what angle convention do you want displayed??